src/services/accounts/AccountsValidateService.ts

```diff
@@ -26,16 +30,21 @@ export class AccountsValidateService extends AbstractService {
 
 		if (defaults.allowedEncodedLengths.includes(u8Address.length)) {
 			const [isValid, , , ss58Prefix] = checkAddressChecksum(u8Address);
+			const networkName = allNetworks.filter(
```

Comment:
Yea for loops will inherently be more performant since no callbacks are in the call stack. But that being said I don't think it's something that will be noticeable here. But is always nice to have. I will leave this up to you, with what you think is cleaner.